pytrip/vdx.py

```diff
@@ -531,7 +531,7 @@ def create_voi_from_cube(cube, name, value=100):
         points = np.zeros((len(contour[0]), 3))
         points[:, 0:2] = contour[0] * cube.pixel_size
 
-        points[:, 2] = i * cube.slice_distance
+        points[:, 2] = i * cube.slice_distance + cube.zoffset
```

Comment:
As a test:
1. Make programmatically in python an empty patient with sperical VOI with negative Z offset, save the files (*.ctx, *.hed, *.vdx)
2. In another program read the generated files (*.ctx, *.hed, *.vdx), check if Z offset is negative and if the data structures are the same as in step 1.

Reply:
Withdraw changes related to adding VOI with z offset, will be added later in new issue